addons/netlify/metadata.yaml

```diff
@@ -0,0 +1,14 @@
+name: netlify
+version: v10.5.1
+description: Interact with Netlify from the comfort of your CLI.
+url: https://github.com/netlify/cli
+icon: https://github.com/netlify/cli/blob/main/cli.png
```

Comment:
Please use `raw.githubcontent` prefix URL for icon. For this is: https://raw.githubusercontent.com/netlify/cli/main/cli.png